armi/bookkeeping/db/__init__.py

```diff
@@ -251,7 +251,7 @@ def _getH5File(db):
     but we have to start somewhere.
 
     .. impl:: The ARMI output file has a language-agnostic format.
-        :id: I_ARMI_DB_H5
+        :id: I_ARMI_DB_H50
```

Comment:
This `impl` tag is on a protected method.

Reply:
Removed. I already have a crumb for this elsewhere anyway.